### PR TITLE
vmm:qemu support config_qemu.toml

### PR DESCRIPTION
**Pull request by HITweihailiuxu**
vmm:qemu support config_qemu.toml

### CODE REVIEW

File: vmm/sandbox/config_qemu_x86_64.toml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+[sandbox]
+log_level = "info"
+
+[hypervisor]
+machine_accelerators = ""
+firmware_path = ""
+cpu_features = "pmu=off"
+cpu_model = "host"
+qemu_path = "/usr/bin/qemu-system-x86_64"
+machine_type = "virt"
```
Comment (xukobe) [inline vmm/sandbox/config_qemu_x86_64.toml:10]: please check this machine type,  I believe it should be 'pc' instread

File: vmm/sandbox/config_qemu_aarch64.toml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+disable_nvdimm = true
+share_fs = "Virtio9P"
+use_vsock = true
+
+[hypervisor.common]
+memory_in_mb = 2048
+vcpus = 1
+kernel_params = "task.log_level=debug task.sharefs_type=9p tsc=reliable rcupdate.rcu_expedited=1 i8042.direct=1 i8042.dumbkbd=1 i8042.nopnp=1 i8042.noaux=1 noreplace-smp= reboot=k console=hvc0 console=hvc1 iommu=off cryptomgr.notests= net.ifnames=0 pci=lastbus=0"
+kernel_path = "/var/lib/kuasar/vmlinux.bin"
+initrd_path = "/var/lib/kuasar/kuasar.img"
```
Comment (xukobe) [inline vmm/sandbox/config_qemu_aarch64.toml:40]: kuasar.img is built as rootfs image in the cargo.toml above, it shouldn't be here as initrd. please either use a real initrd image or put kuasar.img in the image_path.

File: vmm/sandbox/config_qemu_x86_64.toml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+disable_nvdimm = true
+share_fs = "Virtio9P"
+use_vsock = true
+
+[hypervisor.common]
+memory_in_mb = 2048
+vcpus = 1
+kernel_params = "task.log_level=debug task.sharefs_type=9p tsc=reliable rcupdate.rcu_expedited=1 i8042.direct=1 i8042.dumbkbd=1 i8042.nopnp=1 i8042.noaux=1 noreplace-smp= reboot=k console=hvc0 console=hvc1 iommu=off cryptomgr.notests= net.ifnames=0 pci=lastbus=0"
+kernel_path = "/var/lib/kuasar/vmlinux.bin"
+initrd_path = "/var/lib/kuasar/kuasar.img"
```
Comment (xukobe) [inline vmm/sandbox/config_qemu_x86_64.toml:40]: same here

File: vmm/sandbox/config_qemu_x86_64.toml (hunk 1 above)
Comment (HITweihailiuxu) [reply vmm/sandbox/config_qemu_x86_64.toml]: fixed

File: vmm/sandbox/config_qemu_aarch64.toml (hunk 2 above)
Comment (HITweihailiuxu) [reply vmm/sandbox/config_qemu_aarch64.toml]: fixed

File: vmm/sandbox/config_qemu_x86_64.toml (hunk 3 above)
Comment (HITweihailiuxu) [reply vmm/sandbox/config_qemu_x86_64.toml]: fixed

File: vmm/sandbox/config_qemu_aarch64.toml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+msize_9p = 8192
+virtio_9p_direct_io = false
+virtio_9p_multidevs = ""
+enable_iothreads = false
+block_device_driver = "VirtioScsi"
+disable_nvdimm = true
+share_fs = "Virtio9P"
+use_vsock = true
+
+[hypervisor.common]
```
Comment (abel-von) [inline vmm/sandbox/config_qemu_aarch64.toml:35]: Can we put all these configs under [hypervisor] in the toml?

Comment (HITweihailiuxu) [reply vmm/sandbox/config_qemu_aarch64.toml]: fixed

**Review by abel-von — APPROVED** [review]

**Comment by abel-von** [comment]
LGTM

**Pull request merged**